package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "document-manager",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "istanbul": "istanbul cover node_modules/.bin/_mocha --reporter lcovonly -- -- test/*",
+    "seed": "NODE_ENV=test node app/seeders/seed.js",
+    "db-setup": "dropdb --if-exists test_doc_manager && createdb test_doc_manager",
+    "test": "NODE_ENV=test npm run db-setup && npm run seed && npm run istanbul",
+    "start": "NODE_ENV=test npm run seed && nodemon server.js",
+    "coverage": "rm -rf coverage/",
+    "coveralls": "cat coverage/lcov.info | coveralls"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cyrielo/document-manager.git"
+  },
+  "author": "Ologho Cyril Paul",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/cyrielo/document-manager/issues"
+  },
+
+  "homepage": "https://github.com/cyrielo/document-manager#readme",
+
+  "dependencies": {
+    "bcryptjs": "^2.3.0",
+    "body-parser": "^1.15.2",
+    "express": "^4.14.0",
+    "jsonwebtoken": "^7.1.9",
+    "morgan": "^1.7.0",
+    "pg": "^6.1.0",
+    "pg-hstore": "^2.3.2",
+    "sequelize": "^3.24.8"
+  },
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "coveralls": "^2.11.15",
+    "dotenv": "^2.0.0",
```

Comment:
Why is `dotenv` a dev dependency? It's being used in the production environment as well